src/widget/weffectname.cpp

```diff
@@ -28,26 +26,23 @@ void WEffect::setup(const QDomNode& node, const SkinContext& context) {
     }
 }
 
-void WEffect::setEffectSlot(EffectSlotPointer pEffectSlot) {
+void WEffectName::setEffectSlot(EffectSlotPointer pEffectSlot) {
     if (pEffectSlot) {
         m_pEffectSlot = pEffectSlot;
-        connect(pEffectSlot.data(), SIGNAL(updated()),
+        connect(pEffectSlot.data(), SIGNAL(effectChanged()),
```

Comment:
Use new style connect syntax

Reply:
fixed in bc7d2a4622f6850efa387d08fe14927658daab83